crates/wit-parser/src/ast/resolve.rs

```diff
@@ -650,6 +651,9 @@ impl<'a> Resolver<'a> {
                             .insert(value.name.name.to_string(), func);
                         assert!(prev.is_none());
                     }
+                    ValueKind::Static(_) => {
+                        unreachable!("static functions are only supported in resources")
```

Comment:
This should be a `bail!` rather than an `unreachable!` since it indicates an error in the source code.

Reply:
Yes agreed, I will make that change.